mitodl/mitxonline-theme
# feat: removed the sock links

### What are the relevant tickets?
Fixes https://github.com/mitodl/hq/issues/3482

### Description (What does it do?)
Removes the sock links from the legacy studio

### Screenshots (if appropriate):
#### Before
<img width="1438" alt="Screenshot 2024-04-03 at 11 22 20 AM" src="https://github.com/mitodl/mitxonline-theme/assets/88967643/17d5aef9-33f7-4328-94fe-9c69de08fd67">

#### After
<img width="1436" alt="Screenshot 2024-04-01 at 1 13 02 PM" src="https://github.com/mitodl/mitxonline-theme/assets/88967643/2024e0b6-a467-4188-bf5b-14155e4143da">



### How can this be tested?
- Setup mitxonline-theme on your local edx-platform instance.
- Go to studio.
- You wont see any "Looking for help with studio?" drop down.
